src/misc/person.v1.yaml

```diff
@@ -26,12 +26,37 @@ properties:
                       - Schekman, Randy
                       - VijayRaghavan, Krishnaswamy
                       - Li Wenhui
+            given_names:
+                title: Given names
+                description: |
+                    The given names of the person.
+
+                    For example:
+                      - Randy
+            surname:
+                title: Surname
+                description: |
+                    The surname of the person.
+
+                    For example:
+                      - Schekman
         required:
           - preferred
           - index
+          - last
     orcid:
         title: ORCID
         type: string
         pattern: ^[0-9]{4}-[0-9]{4}-[0-9]{4}-[0-9]{3}[0-9X]$
+    correspondence:
```

Comment:
We should make this consistent with https://github.com/elifesciences/api-raml/blob/c3aaefd514b991848c3254cd7a4633c4e64e96b2/src/misc/author.v1.yaml#L36-L40